tinygrad/codegen/lowerer.py

```diff
@@ -115,10 +115,11 @@ def lower(self, ast:LazyOp, opts:Renderer) -> UOp:
     first_upcasted = len(full_shape)-ki.upcasted
     # if there's no reduce, this is first_upcasted
     first_reduce = [x!=y for x,y in zip(ast.src[0].arg.st.shape[:first_upcasted]+(0,), full_shape[:first_upcasted]+(1,))].index(True)
-    local_loads = [x for x in ast.parents if x.op is BufferOps.LOAD and x.arg.idx == -1]
+    local_loads = [x for x in ast.parents if x.op is BufferOps.LOAD and x.arg.idx < 0]
     # NOTE: this is taking the first one...there may be subtlelies here with multireduces
```

Comment:
can you change this comment if it no longer applies and describe what it's doing now?